server/src/compression.rs

```diff
@@ -566,6 +637,110 @@ mod tests {
         )
     }
 
+    /// Function for generating constant/random/linear/random-linear test data with the [ThreadRng](rand::rngs::thread::ThreadRng) randomizer.
```

Comment:
I think it would be best if the style of comments used is consistent throughout the project.

Reply:
Yeah, I think I was following this [convention](https://doc.rust-lang.org/beta/rust-by-example/meta/doc.html) because I couldn't find examples of functions with this many parameters in the project. So should I change this to a single paragraph explaining its purpose, or is there a better example to follow somewhere?

Reply:
There are a few examples of other functions in the project with similar or more parameters, for example [`try_compress`](https://github.com/ModelarData/ModelarDB-RS/blob/master/server/src/compression.rs#L53). The comments for `generate_data` and `generate_timestamps` should be changed.

Reply:
Fixed.